taichi/program/state_flow_graph.cpp

```diff
@@ -139,7 +139,8 @@ void StateFlowGraph::clear() {
   // TODO: GC here?
   nodes_.resize(1);  // Erase all nodes except the initial one
   initial_node_->output_edges.clear();
-  latest_state_owner_.clear();
+  std::fill(latest_state_owner_.begin(), latest_state_owner_.end(),
```

Comment:
I'm a little bit concerned on this linear op because `latest_state_owner_` will always grow now. But I guess if this turns out to be a problem, we can reclaim the assigned unique ids back to IR bank.. (something like the free list)

Reply:
Right, let's stick to it now and until this becomes the bottleneck. I guess this will not be an issue unless a user intentionally constructs an SFG to stress-test the system :-)